Dockerfile

```diff
@@ -44,12 +41,16 @@ RUN cd /opt/ \
   && make install \
   && rm -rf /opt/Python-3.7.4.tgz
 
+# update pip
+RUN pip install --upgrade pip==18.1
```

Comment:
Do we need to use this older version of pip?

Reply:
Can this affect current CI pipelines? (e.g. this image is the one used for all of them right?

Reply:
Otherwise Ansible installation fails.
No, there is no impact, python3 and python2 existing pipelines are running fine.

Reply:
Each ETL installation has its dedicated venv, also.

Reply:
I'm bundling ansible with etl-manager https://github.com/hellofresh/hf-etl-tool/blob/master/setup.py#L20 (just need to **pip** install hf-etl-manager` and it's ready to go)

It is working just fine with python 3.7 and pip 20.1 venv. Can we drop in from the base image maybe?

Reply:
👍 let me try